examples/coupling_edwards_teng/000_coupling.py

```diff
@@ -0,0 +1,99 @@
+import xtrack as xt
+import numpy as np
+import matplotlib.pyplot as plt
+from cpymad.madx import Madx
+import xobjects as xo
+
+mad = Madx()
+mad.call('../../test_data/lhc_2024/lhc.seq')
+mad.call('../../test_data/lhc_2024/injection_optics.madx')
+
+mad.beam()
+mad.use('lhcb1')
+
+mad.globals.on_x1 = 0
+mad.globals.on_x2h = 0
+mad.globals.on_x2v = 0
+mad.globals.on_x5 = 0
+mad.globals.on_x8h = 0
+mad.globals.on_x8v = 0
+
+mad.globals.on_sep1 = 0
+mad.globals.on_sep2h = 0
+mad.globals.on_sep2v = 0
+mad.globals.on_sep5 = 0
+mad.globals.on_sep8h = 0
+mad.globals.on_sep8v = 0
+
+mad.globals.on_a2 = 0
+mad.globals.on_a8 = 0
+
+mad.globals['kqs.a67b1'] = 1e-4
+
+twmad = mad.twiss()
+
+line = xt.Line.from_madx_sequence(mad.sequence.lhcb1, deferred_expressions=True)
+line.particle_ref = xt.Particles(p0c=450e9)
+tw = line.twiss4d(compute_coupling_elements_ed_teng=True)
+
```

Comment:
I would use something more synthetic than `compute_coupling_elements_ed_teng`. For example `coupling_edw_teng`.